ci-operator/config/openshift-priv/cloud-network-config-controller/openshift-priv-cloud-network-config-controller-master.yaml

```diff
@@ -57,6 +57,16 @@ tests:
   commands: make test
   container:
     from: src
+- as: e2e-aws-serial
+  optional: true
```

Comment:
I only meant the ovn version of this job. sorry :(
the openshift-sdn version of this aws-serial job is passing at a much better rate. so if you
want to make it required, it would make sense to me

Reply:
It's me who mis-read your comment. I adjusted it, that makes indeed most sense :-)